Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1673,6 +1673,18 @@ def lhegensim(fragment,howMuch):
                                        }
     if cust!=None : upgradeStepDict['GenSimHLBeamSpotFull'][k]['--customise']=cust
     if era is not None: upgradeStepDict['GenSimHLBeamSpotFull'][k]['--era']=era
+
+    upgradeStepDict['GenSimHLBeamSpotFull14'][k]= {'-s' : 'GEN,SIM',
+                                       '-n' : 10,
+                                       '--conditions' : gt,
+                                       '--beamspot' : 'HLLHC14TeV',
+                                       '--datatier' : 'GEN-SIM',
+                                       '--eventcontent': 'FEVTDEBUG',
+                                       '--geometry' : geom
+                                       }
+    
+    if cust!=None : upgradeStepDict['GenSimHLBeamSpotFull14'][k]['--customise']=cust
+    if era is not None: upgradeStepDict['GenSimHLBeamSpotFull14'][k]['--era']=era
```

Comment:
No, I just did this to make sure a physically consistent beamspot was always used given that the mix of 13 and 14 TeV workflows was already there (just take a look at runTheMatrix.py -w upgrade -n).